run-tnf-container.sh

```diff
@@ -48,10 +48,13 @@ usage() {
 	  -k: set path to one or more local kubeconfigs, separated by a colon.
 	      The -k option takes precedence, overwriting the results of local kubeconfig autodiscovery.
 	      See the 'Kubeconfig lookup order' section below for more details.
+	  -c: set path to one or more local dockercfgs, separated by a colon.
+	      The -c option takes precedence, overwriting the results of local dockercfg autodiscovery.
+	      See the 'DockerCfg lookup order' section below for more details.
```

Comment:
Is this section missing ("DockerCfg lookup order")?

Reply:
Good call, I'll add that section.